src/bin/rendezvous_server.rs

```diff
@@ -20,6 +22,7 @@ struct Cli {
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    init(LevelFilter::INFO, true)?;
```

Comment:
Why not a `--json` flag that controls this?

Reply:
yes please :)